TCAT/Utilities/Network.swift

```diff
@@ -92,7 +92,16 @@ class Network {
         request.method = .get
         return request
     }
-
+    
+    class func getDelay(tripId: String, stopId: String) -> APIRequest<JSON, Error> {
+        let request: APIRequest<JSON, Error> = tron.swiftyJSON.request("delay")
+        request.method = .get
+        request.parameters = ["stopID": stopId, "tripID": tripId]
+        
+        print("Delay request URL: \(source)/delay?stopID=\(request.parameters["stopID"]!)&tripID=\(request.parameters["tripID"]!)")
```

Comment:
Should we log these? Or comment them out for now

Reply:
I havent actually called that yet. Will do that in PR for next week